modules/currency-l0/src/main/scala/org/tessellation/CurrencyL0App.scala

```diff
@@ -0,0 +1,30 @@
+package org.tessellation
+
+import cats.effect._
+
+import org.tessellation.cli.method._
+import org.tessellation.ext.kryo._
+import org.tessellation.schema.cluster.ClusterId
+import org.tessellation.sdk.app.{SDK, TessellationIOApp}
+
+import com.monovore.decline.Opts
+
+final class CurrencyL0App(
+  header: String,
+  clusterId: ClusterId, // TODO: It should be token symbol probably but on the other hand it is hard to make it unique
+  helpFlag: Boolean = true,
+  version: String = "" // TODO: Should versioning be unique per currency?
+) extends TessellationIOApp[Run](clusterId.toString, header, clusterId, helpFlag, version) {
```

Comment:
imo we can create one more trait - `CurrencyApp`, and embed all the things there (inside). in the `CurrencyApp` we will specify just the details (like token name/symbol)